USAGE_android.md

```diff
@@ -260,6 +514,215 @@ originally named `gfxrecon_capture.gfxr` and was created at 2:35 PM
+be to add a delay within the first call(s) to vkMapMemory to attempt to force
+the capture library initialization thread to set its signal handler before the
+capture layer does.
+
+#### Memory Tracking Known Issues
+
+There is a known issue with the page guard memory tracking method.
+The logic behind that method is to apply a memory protection to the
+guarded/shadowed regions so that accesses made by the user to trigger a
+segmentation fault which is handled by GFXReconstruct.
+If the access is made by a system call (like `fread()`) then there won't be a
+segmentation fault generated and the function will fail.
+As a result the mapped region will not be updated.
+
+### Troubleshooting Capturing of Applications
+
+There are several steps to take to determine why a capture may not be working.
+Attempt each of these and re-run after the suggested modifications to see if
+the problem resolves.
+
+#### Enable LogCat
+
+First, enable logcat filtering out specifically messages for GfxReconstruct:
+
+```bash
+adb logcat -s gfxrecon
+```
+
+This should report messages that may be useful in determining the source of the
+capture error.
+
+An extended option would be to look for both `gfxrecon` and `vulkan` messages
+using this command:
+
+```bash
+adb logcat -s gfxrecon,vulkan
+```
+
+#### Stop Your App Before Capture
+
+Sometimes, the application could still be running in the background prior to
+enabling the capture layer.
+In this case, it is best to force stop the application and restart it in order
+for the capture to work properly.
+
+```bash
+adb shell am force-stop ${Application Full Name}
+```
+
+**NOTE:** Replace `${Application Full Name}` with the full Android name of the
+application being captured, for example "com.khronos.vulkan_samples" is the full
+name of the Vulkan Samples.
+
+#### Re-installing The Application
+
+It is entirely possibly, while developing the application that it was uploaded
+to the phone prior to enabling the GfxReconstruct capture layer.
+
+To be sure it is up to date, uninstall the old application, and re-install it:
+
+```bash
+adb uninstall ${Application Full Name}
+```
+
+**NOTE:** Replace `${Application Full Name}` with the full Android name of the
+application being captured, for example "com.khronos.vulkan_samples" is the full
+name of the Vulkan Samples.
+
+Now, re-install the application from the directory where it was built.
+
+##### Forcing Aligned Pages
+
+In some cases, the application may write to buffers that are bound in the same
+mapped page of memory resulting in data being lost when copying pages from the
+page_guard shadow allocation to the real allocation.
+This data loss can result in visible corruption during capture.
+
+To avoid this issue when using `page_guard`, force on page aligned memory
+buffers using the following option:
+
+```bash
+adb shell "setprop debug.gfxrecon.page_guard_align_buffer_sizes 'true'"
+```
+
+#### Enable Page Guard
```

Comment:
This is the default so it doesn't need to be set.

Reply:
Ok, I didn't know page guard was enabled by default.  I'll remove this.